src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -805,6 +839,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                    Me.OptionExplicit = other.OptionExplicit AndAlso
                    Me.OptionCompareText = other.OptionCompareText AndAlso
                    Me.EmbedVbCoreRuntime = other.EmbedVbCoreRuntime AndAlso
+                   Me.SuppressEmbeddedDeclarations = other.SuppressEmbeddedDeclarations AndAlso
```

Comment:
I mentioned offline that I thought other fields weren't using the constructor.  Turns out I was confused because I only looked at the public constructor.  I've update the non-public ones.